packages/client/lib/CredentialRequestClient.ts

```diff
@@ -1,10 +1,10 @@
+import { CredentialRequest, CredentialResponse, OpenIDResponse, ProofOfPossession, URL_NOT_VALID } from '@sphereon/openid4vci-common/lib';
```

Comment:
This is referencing a dir in the package, suggesting the exports are not in order. Should not have to include /lib

Reply:
exports were fine. Intellij auto-refactor upone move added the folder.
Removed the lib part from path.